core-js/src/main/javascript/cdf-legacy/queries/coreQueries.js

```diff
@@ -681,7 +681,7 @@
     interfaces: {
       lastResultSet: {
         reader: function(json) {
-          json = eval("(" + json + ")");
+          json = JSON.parse(json);
```

Comment:
For this, we might need some real unit tests to check if this is failing. For this, we must pick real query results and use them as base